cmd/ipfs/main.go

```diff
@@ -182,6 +182,9 @@ func (i *cmdInvocation) Run(ctx context.Context) (output io.Reader, err error) {
 	if u.GetenvBool("DEBUG") {
 		u.Debug = true
 	}
+	if u.GetenvBool("IPFS_LOW_MEM") {
+		u.LowMemMode = true
```

Comment:
Yeah, my thought was to keep this 'unofficial' until we're more confident about it working. Would suck to have a low memory mode that increased memory for some people, lol